sql/core/src/main/scala/org/apache/spark/sql/execution/Exchange.scala

```diff
@@ -33,7 +32,11 @@ import org.apache.spark.util.MutablePair
  * :: DeveloperApi ::
  */
 @DeveloperApi
-case class Exchange(newPartitioning: Partitioning, child: SparkPlan) extends UnaryNode {
+case class Exchange(
+    newPartitioning: Partitioning,
+    child: SparkPlan,
+    sort: Boolean = false)
```

Comment:
Nit, I would put the child last as we do with most other operators even though this requires a little more refactoring.

Can you also add scaladoc for the parameters? (sorry there is not some there already!)